### PR TITLE
Attendance

### PR DESCRIPTION
**Pull request by Sameerthe-Big-O**
sir i've done all the operations in just one ticket

### CODE REVIEW

File: models/Employee.js [hunk 1]
```diff
@@ -66,9 +67,12 @@ const employeeSchema = mongoose.Schema({
         type: String,
         enum: ['A+', 'A-', 'B+', 'B-', 'AB+', 'AB-', 'O+', 'O-'],
```
Comment (AbubakarZorrain) [inline models/Employee.js:68]: What is this?

File: public/wappalyzer_exercism-org-1680604765144.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
...s","Drupal themes","JavaScript graphics","Operating systems","Maps","Authentication","Cross border ecommerce","Fulfilment","Ecommerce frontends","Rich text editors","Programming languages","Databases","CRM","Cryptominers","Editor","Search engines","CI","Database managers","Documentation tools","Hosting panels","Issue trackers","Webmail","Network services","","Development","Network storage","CMS","Message boards","Ecommerce","Photo galleries","Wikis","Blogs","LMS","Media servers","Remote Access","Feed readers","DMS","Page builder","Accounting","Static site generators","Phone number","Skype","WhatsApp","Email address","Email address (verified)","Email address (safe)","Twitter","Facebook","Instagram","GitHub","TikTok","YouTube","Pinterest","LinkedIn","Owler","Title","Description","Copyright","Copyright year","Responsive","schema.org types","Cert organisation","Cert country","Cert state","Cert locality","Cert issuer","Cert protocol","Cert expiry","SPF record","DMARC record","SSL/TLS enabled","Google Analytics","Google AdSense","Medianet","Facebook","Optimizely","Company name","Inferred company name","Industry","About","Locations","Company size","Company type","Company founded","People"
```
Comment (AbubakarZorrain) [inline public/wappalyzer_exercism-org-1680604765144.csv:1]: Why adding this file? and other png files...?

File: routes/index.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ const unProtectedRouter = express.Router();
 // Un-Protected Routes
 unProtectedRouter.use("/user", userRoute);
 unProtectedRouter.use("/employee", employeeRoute);
-unProtectedRouter.use("/attendence", attendenceRoute);
+unProtectedRouter.use("/attendance", attendenceRoute);
```
Comment (AbubakarZorrain) [inline routes/index.js:21]: Rename it correctly, attendenceRoute. spell mistake.

File: services/employee.js [hunk 4]
```diff
@@ -40,6 +39,7 @@ const EmployeeServices = {
                 image: path,
                 date_of_birth: body.date_of_birth,
                 blood_group: body.blood_group,
+                probation_period: body.probation_period
             };
             // }
```
Comment (AbubakarZorrain) [inline services/employee.js:44]: Remove Comment...

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

File: models/Employee.js (hunk 1 above)
Comment (Sameerthe-Big-O) [reply models/Employee.js:68]: this is the file i'm talking about

**Pull request closed without merging**